frontend/src/components/MailItem.tsx

```diff
@@ -16,6 +16,11 @@ const useStyles = makeStyles({
   mailItem: {
     minHeight: "4rem",
     padding: "0 1rem",
+    marginBottom: "5px",
+  },
+  undecidedMatch: {
+    backgroundColor: "grey",
```

Comment:
It's a little dark, maybe a lighter grey?

Reply:
I agree. Looks a bit too dark.